zerver/openapi/zulip.yaml

```diff
@@ -5846,6 +5846,68 @@ paths:
                     description: |
                       An example JSON error response when attempting to deactivate the only
                       organization owner in an organization
+  /users/me/status:
+    post:
+      operationId: update-self-status
+      summary: Update your status
+      tags: ["users"]
+      description: |
+        Change your status.
```

Comment:
This should link to the /help/ documentation for the feature.

Reply:
Let's just mention status here -- this endpoint isn't used to change availability.

Reply:
Oh, I confused `availability` with `away`. Fixed it.